docs/tutorials/multimodal/object_detection/index.rst

```diff
@@ -30,6 +30,17 @@ Quick Start
       How to train high quality object detection model with MultiModalPredictor in under 5 minutes on COCO format dataset.
 
 
+Quick Start
+------------------
+.. container:: cards
+
+   .. card::
+      :title: Quick Start on a Tiny COCO Format Dataset
```

Comment:
Is this for the Tiny Motorbike dataset? Do we want to add the dataset name in the title